core/types/backwards_compat.libevm_test.go

```diff
@@ -26,91 +26,9 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/ava-labs/libevm/common"
-	. "github.com/ava-labs/libevm/core/types"
-	"github.com/ava-labs/libevm/libevm/cmpeth"
-	"github.com/ava-labs/libevm/libevm/ethtest"
 	"github.com/ava-labs/libevm/rlp"
 )
 
-func TestHeaderRLPBackwardsCompatibility(t *testing.T) {
```

Comment:
This has been effectively cut and paste, without any change, into the other file.